app/views/checkout/thankyou.scala.html

```diff
@@ -1,4 +1,10 @@
-@()
+@import model.GuestAccountData
+@import services.UserId.{paramName => userIdParam}
+@import services.IdentityToken.{paramName => identityTokenParam}
+
+@import routes.Checkout.processFinishAccount
+
+@(formOpt: Option[Form[GuestAccountData]] = None)
```

Comment:
Can we name `formOpt` to be something more related to `GuestAccountData` e.g. `guestAccountForm` ?